.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -61,6 +61,8 @@ export default class Playground extends React.Component<never, State> {
 * [ ] Tested on [mobile](https://github.com/Shopify/polaris-react/blob/master/documentation/Tophatting.md#cross-browser-testing)
 * [ ] Tested on [multiple browsers](https://help.shopify.com/en/manual/intro-to-shopify/shopify-admin/supported-browsers)
 * [ ] Tested for [accessibility](https://github.com/Shopify/polaris-react/blob/master/documentation/Accessibility%20testing.md)
+* [ ] Updated `README.md` with documentation changes
```

Comment:
I think this might be confusing since there's multiple `README.md` files in the repo, can you use language that's bit more specific like `if you modified a component, then update that component's README file` or something like that

Reply:
Updated to specify the component-level `README.md`.